app/js/populateHandlebars.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Populate handlebars template
+ *
+ * @param targetElement the html element to insert handlebars template into
+ * @param handlebarsPath the path to where the handlebars template is located
+ * @param APIpath the API path
+ * @returns {Promise<void>}
+ */
+async function populateHandlebars(targetElement, handlebarsPath, APIpath) {
```

Comment:
if your going to create this as a generic function you need to refactor the other parts of the application that use handlebars templates to use this function, to keep it consistent.